openpype/hosts/max/plugins/load/load_tycache.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from openpype.hosts.max.api import lib, maintained_selection
+from openpype.hosts.max.api.lib import (
+    unique_namespace,
+
+)
+from openpype.hosts.max.api.pipeline import (
+    containerise,
+    get_previous_loaded_object,
+    update_custom_attribute_data
+)
+from openpype.pipeline import get_representation_path, load
+
+
+class TyCacheLoader(load.LoaderPlugin):
+    """Point Cloud Loader."""
+
+    families = ["tycache"]
+    representations = ["tyc"]
+    order = -8
+    icon = "code-fork"
+    color = "green"
+
+    def load(self, context, name=None, namespace=None, data=None):
+        """Load tyCache"""
+        from pymxs import runtime as rt
+        filepath = os.path.normpath(self.filepath_from_context(context))
```

Comment:
Looking at the trace for `self.filepath_from_context(context)` it ends up calling `get_representation_path` which seems to already return a normalized path (if the file exists that is).
```suggestion
        filepath = self.filepath_from_context(context)
```